### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by musangamfure**
Hello, 

In this PR, I have worked on database transactions, modifying, and deleting data in SQL features.

Those are what I have achieved:

- [ ] Add an extra column
- [ ] Inserted more animals
- [ ] Update the  data.sql, schema.sql, queries.sql files

Below are the screenshots for all the queries:

Kindly Check and let me know what you think.

Happy Coding🎉🎉

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,64 @@ SELECT * FROM animals WHERE neutered = TRUE;
...
+
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals WHERE escape_attempts = 0;
+SELECT avg(weight_kg) from animals;
+SELECT * FROM animals WHERE neutered = true ORDER BY escape_attempts DESC LIMIT 1;
+SELECT * FROM animals WHERE neutered = false ORDER BY escape_attempts DESC LIMIT 1;
+SELECT MAX(weight_kg) FROM animals WHERE species = 'pokemon';
+SELECT MIN(weight_kg) FROM animals WHERE species = 'pokemon';
+SELECT MAX(weight_kg) FROM animals WHERE species = 'digimon';
+SELECT MIN(weight_kg) FROM animals WHERE species = 'digimon';
```
Comment (Munsa1) [inline queries.sql:75]: - [ ] (Optional) Again here you could reduce these queries into one by grouping them:

e.g

```
   SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
   GROUP BY species;
```

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,64 @@ SELECT * FROM animals WHERE neutered = TRUE;
...
+SELECT AVG(escape_attempts) FROM animals WHERE species = 'pokemon' and date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
+SELECT AVG(escape_attempts) FROM animals WHERE species = 'digimon' and date_of_birth BETWEEN '1990-01-01' AND '2000-12-31';
```
Comment (Munsa1) [inline queries.sql:77]: - [ ] (Optional) Lastly here again by grouping these two statements we can use one query to get the same answer:

e.g
```
   SELECT species, AVG(escape_attempts) FROM animals
   GROUP BY species
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/12/31';;

```

File: queries.sql [hunk 3]
```diff
@@ -14,4 +14,64 @@ SELECT * FROM animals WHERE neutered = TRUE;
...
+SELECT * FROM animals; -- Verify changes after commit
+
+
+
+
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals WHERE escape_attempts = 0;
+SELECT avg(weight_kg) from animals;
+SELECT * FROM animals WHERE neutered = true ORDER BY escape_attempts DESC LIMIT 1;
+SELECT * FROM animals WHERE neutered = false ORDER BY escape_attempts DESC LIMIT 1;
```
Comment (Munsa1) [inline queries.sql:71]: - [ ] (Optional) Here, you could combine these two queries into one by grouping them using the GROUP BY provided in SQL:

e.g
```
   SELECT neutered, AVG(escape_attempts) FROM animals
   GROUP BY neutered;

```

**Review by Munsa1 — APPROVED** [review]
Hi @musangamfure, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Verified Change Of All Queries :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 
Good Commit Messages :heavy_check_mark: 

## Optional suggestions
- [ ] Check the optional sugestions below


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (musangamfure) [reply queries.sql:71]: Hello @Munsa1,

Thanks for your  suggestions.

**Pull request merged**